### ISSUE TITLE
Kollektivprioritering Skilting feil i kalkulasjoner?

### ISSUE THREAD

**Issue by 7er** [7er/elm-nka#30]
Årlig passasjernytte kalkuleres slik:

=B12*(B14/60)*$Forutsetninger.D12

Hvor B14 er sekunders forsinkelse per kjøretøy. Det virker rart at sekunders forsinkelse av kjøretøy som ikke har prioritet går inn i passasjernytten til busser som får prioritet?

Jeg jobber med den nå og implementer som spesifisert i regneark, men ut i fra hvordan Kollektivprioritering lyskryss er virker det rart. 

Jeg ville tro at noe sånt som å angi avganger per år forbi krysset og så gange med sekunder spart per prioritert kjøretøy og så gange med gjennomsnitts antall passasjerer per avgang ville make mer sense?? for å kalkulere passasjernytten per år

**Comment by nilsfearnley** [comment]
Bra observert. Det er samme nytteelementer som for prioritering ved lyskryss.
Tidsbesparelse for kollektivtransporten skal gi passasjernytte og nytte for operatør
Tidstap for øvrig trafikk gir negativ effekt på "andre"

**Comment by nilsfearnley** [comment]
Jeg har ikke exelarket foran meg, men tror at man legger inn en total, årlig tidsbesparelse (i mnutter) for kollektivtranporten. Dette antall minutter multipliseres med koll.passasjerers tidsverdi for brukernytten, og med operatørkostnad per minutt for operatørnytte.

**Comment by 7er** [comment]
I kollektivpriotering ved skilting er dette anderledes. Her angis det hvor mange avganger som passerer krysset med priortering pr år. Hvis man visste belegget per bussavgang (hadde et felt for dette) så kunne man gange avganger med belegg og så bruke sekunder tidsgevinst per kjøretøy (dette feltet finnes også allerede). 

Det er selvfølgelig mange andre måter å regne på det. Jeg lar tiltaket stå sånn som spesifisert i regnearket, så tar du kontakt når du har anledning til å gå igjennom å rette dette. 

Ok??

**Comment by nilsfearnley** [comment]
OK. Jeg får se på det når jeg har exeldokumentet foran meg. Men du har helt sikkert rett i at det er feil i excel-arket.

Nils

**Comment by 7er** [comment]
Jeg gjetter at de 2 tiltakene skal være likere hverandre. Dvs. at skilting
skal likne mer på lyskryss, de er ikke så like i regnearket

søn. 29. okt. 2017 kl. 15:43 skrev nilsfearnley <notifications@github.com>:

**Comment by 7er** [comment]
Skal vi ta vekk dette tiltaket fra app'en før seminarpresentasjonen på tirsdag? @nilsfearnley

**Comment by nilsfearnley** [comment]
Ja, ta det gjerne ut av kalkulatoren. Ikke superviktig å få det ut før seminaret 7. november, men greit at vi ikke legger ut tiltak med feil i når folk skal være testpilot.

**Comment by 7er** [comment]
Fra dokumentet til @nilsfearnley 

> Dette tiltaket er foreløpig ikke del av Kollektivkalkulator, trolig pga feilen som syver påviste i epost 29/10-17. Syver har rett. Passasjernytten skulle hentet tidsgevinst fra celle B16 (sekunders gevinst per kjøretøy) og ikke celle B14: 
> =B12*(B16/60)*Forutsetninger!D12
> Altså: =B12*(B16/60)*Forutsetninger!D12
>

**Comment by 7er** [comment]
Jeg og @nilsfearnley gikk igjennom dette og det ser ut som tiltaket (både i excel og programkode) beregner feil brukernytte. Tiltaket må gåes mer nøye igjennom i excel før vi kan implementere i program
